WooCommerce/Classes/ViewRelated/Dashboard/Settings/Domains/DomainSettingsViewModel.swift

```diff
@@ -68,10 +68,10 @@ private extension DomainSettingsViewModel {
     func handleDomainsResult(_ result: Result<[SiteDomain], Error>) {
         switch result {
         case .success(let domains):
-            let stagingDomain = domains.first(where: { $0.renewalDate == nil })
+            let stagingDomain = domains.first(where: { $0.isWPCOMStagingDomain || $0.type == .wpcom })
```

Comment:
❓ Should this be named `freeDomain` instead? It seems to me when reading this code that `stagingDomain` should check for `isWPCOMStagingDomain` only, since `type == .wpcom` may include non-staging domains.

Reply:
sure, I renamed it to `stagingOrWPCOMDomain` in https://github.com/woocommerce/woocommerce-ios/pull/8848/commits/8adf56801e73bdbbc7af6fa160f762a089638b36